.gitignore

```diff
@@ -5,4 +5,5 @@
 /openapi.json
 /openapi.yaml
 /docs/.vuepress/dist
-.phpunit.result.cache
\ No newline at end of file
+.phpunit.result.cache
+.idea
```

Comment:
I would suggest keeping `.idea` in your own local global git-ignore.
It gets a mess if you have to start adding ignores for each and every IDE/editor for all contributors.

Reply:
Fair enough - habbit from working mostly on my own projects :)